src/python/CMSSpark/spark_utils.py

```diff
@@ -141,9 +141,11 @@ def file_list(basedir, fromdate=None, todate=None):
 def print_rows(df, dfname, verbose, head=5):
     "Helper function to print rows from a given dataframe"
     if  verbose:
-        print("First rows of %s" % dfname)
+        i = 1
+        print("First " + str(head) + " rows of %s" % dfname)
         for row in df.head(head):
```

Comment:
instead of using explicit i=1 and i=i+1 you may change the for loop to use enumerate function which will provide the index, e.g.
```
for i, row in enumerate(df.head(head)):
```